packages/experimental/TabList/src/TabIndicator/TabIndicator.styling.ts

```diff
@@ -0,0 +1,45 @@
+import type { Theme, UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+
+import { tabIndicatorName } from './TabIndicator.types';
+import type { TabIndicatorTokens, TabIndicatorSlotProps, TabIndicatorProps } from './TabIndicator.types';
+import { defaultTabIndicatorTokens } from './TabIndicatorTokens';
+
+const tabIndicatorStates: (keyof TabIndicatorTokens)[] = ['vertical'];
```

Comment:
Are there supposed to be other states? Like horizontal?

Reply:
There isn't much styling to TabList itself except for its orientation. The `vertical` prop being true will apply the vertical state. Otherwise, horizontal rendering is done by default.